src/main/java/com/autotune/analyzer/serviceObjects/Converters.java

```diff
@@ -73,6 +73,7 @@ public static KruizeObject convertCreateExperimentAPIObjToKruizeObject(CreateExp
                 kruizeObject.setClusterName(createExperimentAPIObject.getClusterName());
                 kruizeObject.setMode(createExperimentAPIObject.getMode());
                 kruizeObject.setPerformanceProfile(createExperimentAPIObject.getPerformanceProfile());
+                kruizeObject.setDataSource(createExperimentAPIObject.getDatasource());
```

Comment:
The datasource column that we have in the `kruize_experiments` table is of type `jsonb`. So, when this kruizeObject is saved in the DB, we either need to make changes in the table to make the column as Varchar or we need to add converter methods to save it as JSON else it will store as null only in the  `kruize_experiments` table.

Reply:
In the kruize_local case we are modifying it to Varchar. See this [file](https://github.com/kruize/autotune/blob/mvp_demo/migrations/kruize_local_ddl.sql)

Reply:
Oh right!. I missed this part. 
Looks fine then since these two are independent of each other.